test/e2e/autoscale_test.go

```diff
@@ -72,20 +73,16 @@ func generateTrafficBurst(clients *test.Clients, logger *logging.BaseLogger, wan
 				} else {
 					logger.Errorf("Nil response. No response headers to show.")
 				}
+				return err
 			}
-			concurrentRequests <- err == nil
-		}()
-	}
 
-	logger.Infof("Waiting for all requests to complete.")
-	got := 0
-	for i := 0; i < want; i++ {
-		if <-concurrentRequests {
-			got++
-		}
+			return nil
+		})
 	}
-	if got != want {
-		return fmt.Errorf("Error making requests for scale up. Got %v successful requests. Wanted %v.", got, want)
+
+	logger.Infof("Waiting for requests to complete.")
```

Comment:
This can be `Info` as nothing is formatted.

Reply:
Right 👍